src/routes/test/test_units.py

```diff
@@ -0,0 +1,18 @@
+from unittest import TestCase
+from fastapi.testclient import TestClient
```

Comment:
Sort Import, remeber [PEP8](https://www.python.org/dev/peps/pep-0008/#imports)
```suggestion
from unittest import TestCase

from fastapi.testclient import TestClient
```